defaults/main.yml

```diff
@@ -40,11 +40,21 @@ jenkins_java_args: ""
 # for example.
 jenkins_java_package: "openjdk-11-jre-headless"
 
-# Location of the JCasC configuration file. Please see the README or JCasC homepage for
+# Location of a single JCasC configuration file. For multiple files, use
+# jenkins_jcasc_config_files instead. Please see the README or JCasC homepage for
 # information about how to generate this file.
-# NOTE: This variable must be defined!
+# NOTE: This variable (or jenkins_jcasc_config_files) must be defined!
```

Comment:
Indeed, I had considered this and my initial implementation deprecated `jenkins_jcasc_config_file`. However, I also wanted to avoid the breaking change. I could go either way, though. Do you think it creates too much confusion to have both variables?